quadlets/foundryvtt/foundryvtt.container

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Foundry VTT Quadlet
+Wants=network-online.target
```

Comment:
User services can't see system services, so `network-online.target` and `local-fs.target` are actually just ignored. Quadlets include a workaround using another service to provide this target. It's on by default, and to disable you must set `DefaultDependencies=false`